app/controllers/accounts/personal_keys_controller.rb

```diff
@@ -40,7 +40,7 @@ def send_new_personal_key_notifications
       end
 
       telephony_responses = MfaContext.new(current_user).
-                            phone_configurations.map do |phone_configuration|
+        phone_configurations.map do |phone_configuration|
```

Comment:
This is moving from 'aligned' to 'indented'.

Purely an opinion thing but I like this better.